client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/package-row.js

```diff
@@ -0,0 +1,165 @@
+				onChange={ abandonHandler } />
+			<span>{ translate( 'Return to sender if package is unable to be delivered' ) }</span>
+		</label>
+
+		<div className="customs-step__restrictions-row">
+			<div className="customs-step__contents-type">
+				<Dropdown
+					id={ packageId + '_contentsType' }
+					title={ translate( 'Contents type' ) }
+					value={ contentsType || 'merchandise' }
+					updateValue={ props.setContentsType }
+					valuesMap={ {
+						merchandise: translate( 'Merchandise' ),
+						documents: translate( 'Documents' ),
+						gift: translate( 'Gift' ),
+						sample: translate( 'Sample' ),
+						other: translate( 'Other…' ),
+					} } />
+				{ 'other' === contentsType && <TextField
+					id={ packageId + '_contentsExplanation' }
+					title={ translate( 'Details' ) }
+					value={ contentsExplanation || '' }
+					updateValue={ props.setContentsExplanation }
+					error={ errors.contentsExplanation } /> }
+			</div>
+
+			<div className="customs-step__restriction-type">
+				<Dropdown
+					id={ packageId + '_restrictionType' }
+					title={ translate( 'Restriction type' ) }
+					value={ restrictionType || 'none' }
+					updateValue={ props.setRestrictionType }
+					valuesMap={ {
+						none: translate( 'None' ),
+						quarantine: translate( 'Quarantine' ),
+						sanitary_phytosanitary_inspection: translate( 'Sanitary / Phytosanitary inspection' ),
+						other: translate( 'Other…' ),
+					} } />
+				{ 'other' === restrictionType && <TextField
+					id={ packageId + '_restrictionExplanation' }
+					title={ translate( 'Details' ) }
+					value={ restrictionExplanation || '' }
+					updateValue={ props.setRestrictionExplanation }
+					error={ errors.restrictionExplanation } /> }
+			</div>
+		</div>
+
+		<TextField
+			id={ packageId + '_itn' }
+			title={
+				<span>{ translate( 'ITN' ) } (<a href="https://pe.usps.com/text/imm/immc5_010.htm"
+					target="_blank" rel="noopener noreferrer">{ translate( 'more info' ) }</a>)</span>
+			}
+			value={ itn || '' }
+			updateValue={ props.setITN }
+			error={ errors.itn } />
+
+		<div className="customs-step__item-rows">
+			<ItemRowHeader />
+			{ uniq( map( items, 'product_id' ) ).map( productId => (
+				<ItemRow
+					key={ productId }
+					productId={ productId }
+					packageId={ packageId }
+					siteId={ siteId }
+					orderId={ orderId } />
+			) ) }
+		</div>
+	</div>;
+};
+
+PackageRow.propTypes = {
+	siteId: PropTypes.number.isRequired,
+	orderId: PropTypes.number.isRequired,
+	packageId: PropTypes.string.isRequired,
+	errors: PropTypes.object,
+	contentsType: PropTypes.oneOf( [ 'merchandise', 'documents', 'gift', 'sample', 'other' ] ),
+	contentsExplanation: PropTypes.string,
+	restrictionType: PropTypes.oneOf( [ 'none', 'quarantine', 'sanitary_phytosanitary_inspection', 'other' ] ),
+	restrictionExplanation: PropTypes.string,
+	abandonOnNonDelivery: PropTypes.bool,
+	itn: PropTypes.string,
+	items: PropTypes.shape( {
+		product_id: PropTypes.number.isRequired,
```

Comment:
`items` is an array, so we need to use `arrayOf` first:

```
items: PropTypes.arrayOf( PropTypes.shape( {
  product_id: PropTypes.number.isRequired,
} ) ).isRequired,
```

Also, let's add the remaining properties of each `item`:

```
items: PropTypes.arrayOf( PropTypes.shape( {
  product_id: PropTypes.number.isRequired,
  height: PropTypes.number,
  length: PropTypes.number,
  name: PropTypes.string,
  product_id: PropTypes.number,
  quantity: PropTypes.number,
  url: PropTypes.string,
  weight: PropTypes.number,
  width: PropTypes.number,
} ) ).isRequired,
```